mcp/mcptools/qiskit_mcp.py

```diff
@@ -0,0 +1,230 @@
+from typing import Any
+import httpx
+from mcp.server.fastmcp import FastMCP
+#from qiskit_ibm_catalog import QiskitFunctionsCatalog
+import json
+ 
+# Initialize FastMCP server
+mcp = FastMCP("qiskitmcp")
+
+@mcp.tool()
+async def run_iskay_optimizer(problem: dict[str, float], problem_type: str, backend_name:str, instance: str, options: dict[str, Any]) -> str:
+    """Quantum Optimizer that solves Quadratic Unconstrained Binary Optimization and higher-order (HUBO) optimization problems
+    The running status of the optimizer is returned by get_optimizer_job_status.
+    The result of the optimizer is returned by get_optmizer_result 
+
+    Args:
+        problem: Coefficients of the optmization problem encoded in a dictionary as follows: { "()" : A, "(i,): Bi, "(i,j)": Cij, ....} The keys of the dictionary must be strings containing a valid tuple if non-repeating integers.
+        problem_type: spin for cost function written in Ising fomulation or binary for cost function written in QUBO/HUBO formulation
+        backend_name: name of backend
+        instance: cloud resource name
+        options: options to handle hardware submission. format is dictionary.  shots: nunber of iteration, num_iterations: total number of Bias Field iterations, use_session: where to run within a session    
+    Ouput:
+        job_id: job id of the optimization job
+    """
+    # catalog = QiskitFunctionsCatalog()
+ 
+    # Access Function
+    # optimizer = catalog.load("kipu-quantum/iskay-quantum-optimizer")
+
+    arguments = {
+        "problem": problem,
+        "problem_type": problem_type,
+        "instance": instance,
+        "backend_name": backend_name,  # such as "ibm_fez"
+        "options": options,
+    }
+ 
+    # job = optimizer.run(**arguments)
```

Comment:
I see so this is commented. Maybe because you cannot run now. Could we setup a mock and enable real when we can with some flag?